configs/benchmark-p2p.json

```diff
@@ -0,0 +1,127 @@
+{
+    "description": "aws 300 nodes, 4 shards",
+    "client": {
+      "num_vm": 0,
+      "type": "m5a.large",
+      "regions": "iad,sfo,cmh,pdx"
+    },
+    "leader": {
+      "num_vm": 1,
+      "type": "c5.large",
+      "regions": "iad,sfo,cmh,pdx",
+      "protection": false,
+      "root": 10
+    },
+    "explorer_node": {
+      "num_vm": 1,
+      "type": "c5.large",
+      "regions": "iad,sfo,cmh,pdx",
+      "root": 25,
+      "userdata": "userdata-systemd-static-exp.sh",
+      "protection": false
+    },
+    "bootnode": {
+       "enable": true,
+       "server": "52.40.84.2",
+       "name": "b6.harmony.one",
+       "port": 9842,
+       "user": "ec2-user",
+       "key": "oregon-key-benchmark.pem",
+       "p2pkey": "b2-9870.key",
+       "log_conn": false
+    },
+    "bootnode1": {
+       "enable": false,
+       "server": "54.86.126.90",
+       "name": "b5.harmony.one",
+       "port": 9845,
+       "user": "ec2-user",
+       "key": "virginia-key-benchmark.pem",
+       "p2pkey": "b1-9870.key",
+       "log_conn": false
+    },
+    "azure": {
+      "num_vm": 0,
+      "regions": [
+        "eastus",
+        "westeurope",
+        "southeastasia"
+      ]
+    },
+    "benchmark": {
+      "shards": 4,
+      "duration": 120,
+      "dashboard": false,
+      "crosstx": 75,
+      "attacked_mode": 0,
+      "peer_per_shard": 75,
+      "minpeer": 8,
```

Comment:
you need to set the minpeer higher to bootstrap the network. Please set it to 51.

Reply:
2/3 * 75 + 1